src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -128,6 +129,16 @@ object ProjectFortis extends App {
           )
         )
       ),
+      "bing" -> List(
+        ConnectorConfig(
+          "BingPage",
+          Map(
+            "accessToken" -> System.getenv("BING_ACCESS_TOKEN"),
+            "searchInstanceId" -> System.getenv("BING_SEARCH_INSTANCE_ID"),
+            "keywords" -> "isis|attack"
```

Comment:
Good comment. These values are just here for testing purposes for now so they're fine to be hard-coded (see also all the other hard-coded values in this section). We'll get rid of them soon and in production, we'll populate the configs from Cassandra which is what Kevin is about to implement.